### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
- Add new screenshots
- Add more informations about contribution

Some things is not perfect, you can improve the file with pleasure :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,88 @@
-# Exodus - Android Application
-
-[![Build Status](https://github.com/Exodus-Privacy/exodus-android-app/actions/workflows/main.yml/badge.svg?branch=master)](https://github.com/Exodus-Privacy/exodus-android-app/actions/workflows/main.yml) [![Language grade: Java](https://img.shields.io/lgtm/grade/java/g/Exodus-Privacy/exodus-android-app.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Exodus-Privacy/exodus-android-app/context:java)
+<h1 align="center">Exodus - Android Application</h1>
```
Comment (pnu-s) [inline README.md:1]: **suggestion**: Not a big fan of using html here, if we can keep using markdown syntax I think that would be easier to maintain

File: README.md [hunk 2]
```diff
@@ -1,19 +1,88 @@
-# Exodus - Android Application
-
-[![Build Status](https://github.com/Exodus-Privacy/exodus-android-app/actions/workflows/main.yml/badge.svg?branch=master)](https://github.com/Exodus-Privacy/exodus-android-app/actions/workflows/main.yml) [![Language grade: Java](https://img.shields.io/lgtm/grade/java/g/Exodus-Privacy/exodus-android-app.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Exodus-Privacy/exodus-android-app/context:java)
```
Comment (pnu-s) [inline README.md:3]: **question**: Why did you remove the badges?

File: README.md (hunk 1 above)
Comment (Jean-BaptisteC) [reply README.md:1]:
```suggestion
<p align="center">
# Exodus - Android Application
</p>
```

Is it better like this, i don't have find other way to centered text in markdown

Comment (codeurimpulsif) [reply README.md:1]: Yes, HTML is the only option I think

Comment (pnu-s) [reply README.md:1]: I don't really like using the HTML format here, but I have to admit it looks good, so why not ;)

**Review by codeurimpulsif — APPROVED** [review]
